app/src/components/SettingsModal/SettingsModal.tsx

```diff
@@ -50,8 +51,8 @@ const SettingsModal: React.FC<SettingsModalState> = ({updateViewState}) => {
                             scrollbar-thin scrollbar-thumb-gray-400 scrollbar-track-gray-200 dark:scrollbar-track-gray-700">
                 <div className="justify-center text-center py-1">
                     <label className="relative inline-flex items-center cursor-pointer">
-                    <input type="checkbox" value="" checked={isLocationSharing} onChange={updateUserSettings}/>
-                    {/* <div className="w-11 h-6 bg-cyan-200 peer-focus:outline-none peer-focus:ring-4 peer-focus:ring-cyan-300 dark:peer-focus:ring-cyan-800 rounded-full peer dark:bg-cyan-600 peer-checked:after:translate-x-full peer-checked:after:border-white after:content-[''] after:absolute after:top-[2px] after:left-[2px] after:bg-white after:border-gray-300 after:border after:rounded-full after:h-5 after:w-5 after:transition-all dark:border-gray-600 peer-checked:bg-cyan-500"></div> */}
+                    <input type="checkbox" value="" checked={isLocationSharing} onChange={updateUserSettings} className="sr-only peer"/>
```

Comment:
Could we remove sr-only peer on checkbox for isLocationSharing in Settings Modal? As of till now, we haven't been adding any screen reader support or aria variables to properly support that. I'd rather just wholistically not support screen readers atm until we decide to support them officially, as we can't track specifically which elements are missing proper aria vars and accesibility support. Not in scope. Furthermore, why are we adding peer onto the checkbox as I don't think we are passing any of its pseudo states down to sibling HTML elements.